beaker.gemspec

```diff
@@ -40,6 +40,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'inifile', '~> 2.0'
   s.add_runtime_dependency 'rsync', '~> 1.0.9'
   s.add_runtime_dependency 'open_uri_redirections', '~> 0.2.1'
+  s.add_runtime_dependency 'in-parallel', '~> 0.1.8'
```

Comment:
Anything prior to 0.1.8 would not work, I suppose that means I should update it to 0.2.0 since there were breaking changes.